.github/ISSUE_TEMPLATE/discussion.yaml

```diff
@@ -0,0 +1,32 @@
+name: Discussion
```

Comment:
If i would ask a question at an announcement it would be related to the announcement. I think we should keep the issues (question/discussions) separate that are not related to the announcement.

Reply:
Alternatively we could also do something like this

Issues are not a place to go ask support questions or start discussions.
Please ask support questions or start discussions on the mailing list or IRC (#freetube:matrix.org) (Web | Matrix)

I also see this allot on other repos

Reply:
Hey, are you going to remove the template?

Reply:
Hi, this is still undecided whether we keep or delete them.

Reply:
We can keep this template and it can be archived if it is decided that discussions will be removed from issues.

Reply:
I'm still in favor of removing this in favor of the discussions tab.

A quick reminder that any user can create a thread as a discussion, so threads wouldn't have to be limited to the announcement threads that we create. We can also mark discussions as answered which would help others looking for something similar.

If this is the direction that we go, then this template along with the question template would simply be deleted.

Reply:
Ah yes didnt even think about it like that. I will remove the templates in next commit!